spec/controllers/api/schedules/schedules_index_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+describe Api::SchedulesController do
+
+  include Devise::TestHelpers
+
+  describe '#index' do
+
+    let(:user) { FactoryGirl.create(:user) }
+
+    it 'lists all schedules for a user' do
+      sign_in user
+      schedules = FactoryGirl
+                    .create_list(:schedule, 2, user: user)
+                    .map(&:_id)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.